.travis.yml

```diff
@@ -9,10 +9,16 @@ addons:
-script: mvn install -Ptravis && mvn javadoc:javadoc
+    - secure: pWproZmuKypqXbd1HjKHWu3CdPhC8i5ghIxpV2yQApM1/Uc151NvRJYzu5L2AQ2pzQ35B90i2uQApJ19dw6Ec2iqvXqmJ2OpWgxD1/eqnVZThB1aQy8PWfKQauoVSBsH1EoD0oiHRmDIZDxR3fSWPikFijLo4MUr8WpwG62RE9JC0PTu50qdIDiS/lSIHoiiH3e1nwNEpXLMjQBZiS7GzzcedZVhF4LEjcdrAl4j/QE4aDksDf33yoLbxQOK+69PYg4Fmw4Ga7WA7G+zL8TM75ok9/bAEyCZ1J2B+scx5HrLK6fNhaAluuLtyn/S3Kl2OP5Uc05/KbJsRKr2bkj4oeFy66B3T8x7e62nAWOPn2tPwdPpJ46gmhbaOuvzHLsEnc0I10ywTjh5zMVMsViNlQ29xGnTP9ipjPZMHCOvb4dUqwUKMBBcEZz02mU/m+XzHFKWIQajYepLkaivEVEaJegseDKlQWFdmtzAUm8mQQufXazlQkcU3J5vo0dsyJNcJmF66vNmZM4OoeUjBHzf166jMp4Al9AgZvAtNxs2+RgbXc8d4jHcWJWlFpCv9X3DeYXdvix3eP/aN9OCsG7n0E8MtWMSi0WBFNYUx0EVncFXazz2La3oFpoHokY+4iae4wk2VCVJdJyB0wTv/2inyqIRBlHzSfI0dFnYzm6dXcM=
+    - secure: cm+dZeYYXiysTp7b5qSBnJ6JmMoKL7On/txWqPkN1ZJmOpyfZO9P1mHNMNKjqz92ie9uA41sLk6+ww0IiXbPlAOvY+S0WkodXgszfrzZq9O/fKnGCzmNAHeAGHiEHrJPH+sKhVuvIy37YASUS7QbCg9opeYDyX5wE374I/WtovAhKYGk1ESr2ziRoC5K+p8fjr8BU1D/30K3eBfPJiI6pd2lZQvq+rgW67zsOVKFtkFLzdKe+WYnM2l3rsmkt+zHkXan6vmYC8YJaR0t6eBYjsIGyFwx4/hQfWR8pkeTCtUGcMWAwLXQ1vBWLBF6hvBHNDU/cm83NasD65ER4R0lU7IKRGIR5rvLWmKbSY/WWB87m1Nx4I3LsK4WFGpj50cw2Nwm77Z78ILjvCwoutP605NrKjdKd8ls+mX2oHFerCaFpnx6bR8io/yzuguc+Dm/ydMaJ4LEESCwS9jUPQBJXBrMNwOsyv0fKHBv0E/MW5dTbQifG8u/UVUWsWGtVspNEIlqEl1D2tR9B8No14y3+LJGj3mDcj6OXM1ImfWqpsfZA/gVEV7d0BxHChQ1DtFTsJ7dsW0uDoV6HJ2vwnJxrkV0fMzuvpjcuX02E2WkL8Xd5ASCv4zYncQjfws6iRkXR/qHDzNWLktugqvO3jcMwZ5sDaxsjUwi9sF0aVhKrvo=
+    - secure: gtowrE44d6hqY30ckpF4Qbd7LeTPQ1VuCJmfZJlYOmSg4KN554+iE++KPPcW0f9LxfxDqOripFPDlKDmwWpnztdkKobvKuqVtyAhB0/9htNi1LtAhAQh6fOUmoZvT4Sd3fh/NHDCrMINzfYjnbRNYXd41HNVyp/ciy2DrHbnSf0u9Zq3clPe8H9ZcaRTFZDdc8enwVl0/eiTNrAG+24dZF51owmZe/81wS4VZmhOeUshQ52xhV4CDF18+hRqYSlFCghWk/a/+dq8xCaIaHpoaXQ1LuvD+qbwGAHC5QBFB8Fo9RCbGBI67mey3YmAoWLj1Q/ZghIL5s+pYs5pdu8J3XaIX2di4WGJnNgdDtQl5yrwjuO75i9y1n5CsLNriLDDpJcLqpURFRI9nFVxQJ7sS12mQ9+BRO9r7KqODUh2dw8rKkF56S7Tbo9MwhbtVMwhcVMxNRv2z2rx99LIvrBbiEhI334nMI/28lvvyQg6VxcolZYb1uzjt3tiejPpnkXwrPIQ1t/P1OSZXO7RWhKmDe296ewuWotiWYFZqbch5v42xlM4InvDTlQ9rRK4i/J/ZRsaDF38Ef53f1Yv94ueKGz2eVGTVDR8jB0F53gJ3orLJMBGLUhmngCXaQL1MYgDIezhu99INUv8i/hdllpvSRq5ppA9mS/H3OBNghQqf+c=
+    - secure: dTfNDVvqeZSx+hlQahSc3Ft3H/48e4q8NsfNuHE4Vxal44N1XvdgUcc3TGJSMCNx7lyp2jIR8ezLRyVnh8heQ7iCTxP0eQgNW9fZJygbnqy9Ar7j9nYUkaSSzAxTSFGR+sFVlRHM98vKoeYGMxp6+h4F7Krzfu7vDeJYXTixwh887Yb/hil9yOvRKqQ2zZ2wKxd8Z4QKbcf1pjRbgNsiK6FVHFZziouBWjJI36Eu/vr2NT0gSHA1Zio9LvV935bImrxf3hAronfPIz+K8q8CEfbcEaeLcfOFkzh0apRhPXWrwqpC6+eApB6S7gcIgvmZhjAQczME6bpOWdLxdGSFaw2c0FJCaDHVRtlPRgt8EuHnrVslxkqQqGR5gIRPBA2zI/cHP6upQGYsPRXpG0zmDnx1S2mS/1w6qHQnF/Ww1c7vbR2hBg8RO4jwl/jeuYe6Yr9CvjsCj7OoAk2CoFLByhl55gN5I28KNsCvA+jecS5hwxcHp/IAFsdj6mWlN+H94NOv6F2Ncod84a6DlbxpaV9rumCM0D6/GWe/XnGy8rKV4fKyUOb+YqlKKO3T+RzuYoeXpHes/GPj3vRsrtIJImov6aJ+zmz1/fMjAkUAzhDjoSRFAgo1Czu/YunlSP0UDQTAA6/uw0w+8slJIW9bPl5hVVFidYDgTBssbKD/rQg=
+    - MAVEN_OPTS="-XX:+TieredCompilation -XX:TieredStopAtLevel=1"
+install: mvn -q install -P quick,travis,!build-extras -B -V
```

Comment:
If the quick profile is specified, the build-extras profile will not run (since it is marked as activeByDefault). I've removed the reference to build-extras on this line to avoid ambiguity.